cmake/add_third_party.cmake

```diff
@@ -35,7 +35,14 @@ macro(rythe_add_third_party)
 
 			foreach(target ${RYTHE_ADD_THIRD_PARTY_FOLDER_TARGETS})
 				if (TARGET ${target})
-					set_target_properties(${target} PROPERTIES FOLDER ${RYTHE_ADD_THIRD_PARTY_FOLDER})
+					string(REGEX REPLACE "^third_party" "4 - third party" filter ${RYTHE_ADD_THIRD_PARTY_FOLDER})
```

Comment:
Is this a quick fix/hack or the only way of doing this?

Reply:
only way i could find..